src/test/java/outcomes/process_executor/go/no_find_executable_file_by_file_path/NoFindExecutableFileByFilePath.java

```diff
@@ -0,0 +1,27 @@
+package outcomes.process_executor.go.no_find_executable_file_by_file_path;
+
+import org.hyperskill.hstest.common.FileUtils;
+import org.hyperskill.hstest.dynamic.input.DynamicTestingMethod;
+import org.hyperskill.hstest.testcase.CheckResult;
+import org.hyperskill.hstest.testing.TestedProgram;
+import outcomes.base.ContainsMessage;
+import outcomes.base.UserErrorTest;
+
+
+public class NoFindExecutableFileByFilePath extends UserErrorTest {
```

Comment:
I am trying to use this piece of code but I am getting an error: `No tests found for given includes: [outcomes.process_executor.go.no_find_executable_file_by_directory.NoFindExecutableFileByDirectory](--tests filter)`

Code:
```java
package outcomes.process_executor.go.no_find_executable_file_by_directory;

import org.hyperskill.hstest.dynamic.DynamicTest;
import org.hyperskill.hstest.testcase.CheckResult;
import org.hyperskill.hstest.testcase.TestCase;
import org.hyperskill.hstest.testing.TestedProgram;
import org.junit.Assume;
import org.junit.BeforeClass;
import outcomes.base.ContainsMessage;
import outcomes.base.UserErrorTest;

import java.util.Arrays;
import java.util.List;

import static org.hyperskill.hstest.testing.ExecutionOptions.includeProcessTesting;


public class NoFindExecutableFileByDirectory extends UserErrorTest<String> {

    @BeforeClass
    public static void stopProcessTest() {
        Assume.assumeTrue(includeProcessTesting);
    }

    @ContainsMessage
    String m1 =
            "Error in test #1\n" +
                    "\n" +
                    "Cannot find a file to execute your code in directory \"";

    @ContainsMessage
    String m2 = "no_find_executable_file_by_directory";

    @Override
    public List<TestCase<String>> generate() {
        return Arrays.asList(
                new TestCase<String>().setDynamicTesting(() -> {
                    TestedProgram main = new TestedProgram(
                            "outcomes.process_executor.go.non_existent_directory");
                    return CheckResult.correct();
                })
        );
    }
}
```

Maybe you know why it doesn't work?

Reply:
Yep, it should be like that since you haven't passed `-DincludeProcessTesting=true` flag to the JVM. To test this class you need to add this flag otherwise JUnit will skip it. You can add it in Run/Debug configuration.

![image](https://user-images.githubusercontent.com/20189630/178734282-fc3cb530-e39c-4073-94d9-8f587cf3fe13.png)